pliant/README.md

```diff
@@ -0,0 +1,133 @@
+
+![curl][12]
+
+![select key][13]
+
+Your endpoint looks like this: ***https://<YOUR_PLIANT_INSTANCE>/api/v1/trigger/<YOUR_PLIANT_USERNAME>/User/<PATH_TO_WORKFLOW>/<WORKFLOW_NOW>?sync=true&api_key=<YOUR_API_KEY>***
+
+![endpoint][14]
+
+Copy this endpoin, starting with ***https*** and ending with the full API Key. Do not include any quotes.
+
+#### Datadog setup
+1. Open Datadog and from the left sidebar, click to **Integrations** > **Integrations**.
+![integrations][15]
+
+2. Enter "webhooks" in the search bar and click on the **webhooks** entry to reveal a configuration window.
+![webhookSearch][16]
+
+
+3. Scroll to "webhooks". Click **New** to add a new webhook to link to the Pliant workflow. First, give the webhook a name in the "name" field. This example uses the name *RestartHost*.
+![webhooksConfig2][17]
+
+Now, paste the URL copied from step 4. For example: 
+
+***https://<YOUR_PLIANT_INSTANCE>/api/v1/trigger/<YOUR_PLIANT_USERNAME>/User/<PATH_TO_WORKFLOW>/<WORKFLOW_NOW>?sync=true&api_key=<YOUR_API_KEY>***
+
+Paste this into the ***URL*** field of the webhook form.
+
+![webhookForm][18]
+
+The request payload is pre-configured. Check the "ENCODE AS FORM" box and click save.
+
+Add this integration to any alert notification in Datadog by adding the recipient of `@webhook-RestartHost`. When the monitor triggers an alert, the webhook triggers your Pliant workflow, and the input variables are sent to Pliant from Datadog.
+
+## Data Collected
+
+### Metrics
+
+The Pliant integration does not provide metrics.
+
+### Service Checks
+
+The Pliant integration does not include any service checks.
+
+### Events
+
+The Pliant integration does not include any events.
+
+## Troubleshooting
+
+Need help? Contact [Datadog support][7].
+
+[1]: https://pliant.io/
+[2]: https://docs.datadoghq.com/agent/kubernetes/integrations/
+[3]: https://github.com/DataDog/integrations-core/blob/master/pliant/datadog_checks/pliant/data/conf.yaml.example
+[4]: https://docs.datadoghq.com/agent/guide/agent-commands/#start-stop-and-restart-the-agent
+[5]: https://docs.datadoghq.com/agent/guide/agent-commands/#agent-status-and-information
+[7]: https://docs.datadoghq.com/help/
+[8]: https://www.pliant.io/wp-content/uploads/2020/05/step1.png
+[9]: https://www.pliant.io/wp-content/uploads/2020/05/step2.png
+[10]: https://www.pliant.io/wp-content/uploads/2020/05/step1-a-.png
+[11]: https://www.pliant.io/wp-content/uploads/2020/05/expand.png
+[12]: https://www.pliant.io/wp-content/uploads/2020/05/curl.png
+[13]: https://www.pliant.io/wp-content/uploads/2020/05/selectDDkey.png
+[14]: https://www.pliant.io/wp-content/uploads/2020/05/endpoint.png
+[15]: https://www.pliant.io/wp-content/uploads/2020/05/integrations_.png
+[16]: https://www.pliant.io/wp-content/uploads/2020/05/webhook_Search.png
+[17]: https://www.pliant.io/wp-content/uploads/2020/05/webhooksConfig3.png
+[18]: https://www.pliant.io/wp-content/uploads/2020/05/webhookForm.png
```

Comment:
Could we move these images into an `/pliant/images/` directory within this repo and replace the links with raw github versions?
See example: https://github.com/DataDog/integrations-extras/tree/master/k6/images

The externally linked images may not be rendered correctly in our Integrations documentation pages which are generated from the README (https://docs.datadoghq.com/integrations/k6/). This would also be easier to maintain in the future :)

Reply:
I have added the pliant/images directory in our forked repo and updated the raw image file paths to correspond to the example given. Please verify the addition of the images folder in this pull request along with the updated paths. Images should populate in README upon merge, as file paths will exist upon merge.